backend/src/routes/api/integrations/nim/nimUtils.ts

```diff
@@ -70,7 +70,7 @@ export const createNIMSecret = async (
   fastify: KubeFastifyInstance,
   namespace: string,
```

Comment:
Missed this before -- lets not pass the namespace -- get it out of fastify in the function.

Reply:
Updated